ngrok/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3b096ceb-d7a5-4bb5-bf0a-3a07d308d56a",
+  "app_id": "ngrok",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Visualize valuable application insights with ngrok HTTP events.",
+    "title": "ngrok",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "ngrok HTTP request events overview dashboard",
+        "image_url": "images/dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "ngrok services platform",
+        "image_url": "images/diag1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "ngrok + datadog",
+        "image_url": "images/diag2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Developer Tools",
+      "Category::Cloud",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Queried Data Type::Logs"  
```

Comment:
Please include a submitted data type if you are sending data to Datadog

Reply:
As seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags)

Reply:
submitted data type added